### ISSUE TITLE
Code completion does not work on Linux

### ISSUE THREAD

**Issue by shyimon** [benfry/processing4#734]
## Description
Code completion is not able to create the panel to show the suggestions

## Expected Behavior
The panel opens with the relevant suggestions

## Current Behavior
Nothing happens and the following error is logged in the console:
`java.lang.NullPointerException: Cannot invoke "javax.swing.ImageIcon.getIconWidth()" because "processing.mode.java.CompletionPanel.classIcon" is null
	at processing.mode.java.CompletionPanel.<init>(CompletionPanel.java:141)
	at processing.mode.java.JavaTextArea.showSuggestion(JavaTextArea.java:520)
	at processing.mode.java.JavaTextArea.lambda$fetchPhrase$0(JavaTextArea.java:332)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:771)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:716)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:741)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)`

## Steps to Reproduce

1. Enable Code completion with Ctrl + Space in Preferences
2. Type `void setup() {
  backg`
3. Press Ctrl + Space
4. See console for error log

## Your Environment
* Processing version: 4.2
* Operating System and OS version: EndeavourOS, Arch Linux 6.3.8

**Comment by benfry** [comment]
Are you using the version from the Processing site (or the releases page here on GitHub), or one that was packaged by someone else?

**Comment by manusaurio** [comment]
I'm not a EndeavourOS user, but I do use Arch Linux and haven't had this problem, the auto completion feature works fine for me. I can tell it's  not a _Linux_ issue.

As far as I know EndeavourOS users also use the AUR repositories. My version of Processing is 4.2, [its latest PKGBUILD on the AUR](https://aur.archlinux.org/cgit/aur.git/plain/PKGBUILD?h=processing) has OpenJDK 17 and ffmpeg as dependencies. The package makes sure Processing uses this version of Java and not the one set by `archlinux-java`, so it should work fine if Processing is being launched in a conventional way.

On the other hand, I've used the GitHub version (not [the outdated one in the AUR](https://aur.archlinux.org/packages/processing-git)) a few times within the present year and haven't had any issues like this either. I just tested it just in case and GitHub's works fine for me, I did the build with [OpenJDK 17.0.7](https://archlinux.org/packages/extra/x86_64/jdk17-openjdk/).
